x-pack/plugins/monitoring/public/components/license/index.js

```diff
@@ -10,10 +10,16 @@ import {
   EuiPageBody,
   EuiSpacer,
   EuiCodeBlock,
-  EuiPanel
+  EuiPanel,
+  EuiText,
+  EuiTextAlign,
+  EuiLink
 } from '@elastic/eui';
 import { LicenseStatus, AddLicense } from 'plugins/xpack_main/components';
 import { FormattedMessage } from '@kbn/i18n/react';
+import chrome from 'ui/chrome';
+
+const licenseManagement = `${chrome.getBasePath()}/app/kibana#/management/elasticsearch/license_management`;
```

Comment:
cc @elastic/kibana-platform 

Is this the appropriate way to cross-link between plugins currently (in legacy land)?

Reply:
@igoristic I'd time box and answer from above ^^ License management doesn't have any licensing constraints (which might usually be a factor in being able to guarantee this kind of linking always works for all users) so I think it's fine as-is